README.md

```diff
@@ -0,0 +1,158 @@
+
+  <h3><b>Hello-Microverse Readme</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
[OPTIONAL] Please add a live demo section you will remove the FAQ section because this FAQ is only an optional section so kindly add the live demo section :slightly_smiling_face: